Parsers/Alsid/afad_parser.kql

```diff
@@ -0,0 +1,46 @@
+// Usage Instruction :
```

Comment:
Parser should follow the standard format, example from this - https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/PulseConnectSecure/PulseConnectSecure.txt.  Need to get a Wiki update to include required template for parsers, but you can use the PulseConnectSecure version as a template.